packages/core/src/editor/BlockNoteSchema.ts

```diff
@@ -66,6 +66,8 @@ export class BlockNoteSchema<
      * A list of custom Styles that should be available in the editor.
      */
     styleSpecs?: SSpecs;
+
+    trailingNode?: boolean;
```

Comment:
why is this added here?

Reply:
It was necessary for the 'typescript' warning, but with 'getBlockNoteExtensions', it's no longer needed, so I removed it in the new commits.